app/ForkedStar.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import { env } from "@/env.mjs";
+import { useEffect, useMemo, useState } from "react";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+
+function ForkedStar(props: any) {
+  const [data, setData] = useState<any>();
+  let forkCount = 0;
+  let starCount = 0;
+
+  useMemo(async () => {
+    const res = await fetch(
+      `https://api.github.com/orgs/${env.NEXT_PUBLIC_GITHUB_ORG}/repos`,
+    );
+    setData(await res.json());
+  }, []);
+
+  data?.map((ls: any) => {
+    if (ls.forks) {
+      forkCount += ls.forks;
+    }
+    if (ls.stargazers_count) {
+      starCount += ls.stargazers_count;
+    }
+    // console.log(ls.full_name, ls.forks, ls.stargazers_count);
+  });
+  console.log(forkCount);
+
+  return (
+    <div className="flex justify-center">
+      {forkCount > 0 && starCount > 0 ? (
+        <>
+          <div className="flex items-center justify-center bg-violet-950 m-4 p-2 w-6/12 rounded-md border-2 border-slate-400 ">
```

Comment:
Let's not use colors outside `secondary` / `primary`. Different organizations deploying leaderboard would have different themes, and the CSS Variables secondary and primary uses the intended theme colors.